libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -134,4 +138,19 @@ describe('AccordionItemComponent', () => {
       expect(spectator.component.toggle.emit).not.toHaveBeenCalled();
     });
   });
+  describe('With list', () => {
```

Comment:
Nitty gritty 😬 But I think we generally do like this
```suggestion
  describe('with list', () => {
```